notebooks/02_image_downloader.ipynb

```diff
@@ -0,0 +1,318 @@
+{
```

Comment:
<p>can we clear the warning in the docs or does this not appear?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>i can clear the outputs so it does not appear in the docs</p><p> </p>